src/reducers/index.js

```diff
@@ -18,12 +16,13 @@ const inputPropertySelector = (state, props) => {
   return items
 }
 
-export const getProperty = createSelector(
-  [ inputPropertySelector ],
-  (propertyURI) => {
-    return propertyURI.items
-  }
-)
+// TODO: Renable use of reselect's createSelector, will need to adjust
+// individual components InputLiteral, InputLookupQA, InputListLOC, etc.,
+// see https://github.com/reduxjs/reselect#sharing-selectors-with-props-across-multiple-component-instances
+export const getProperty = (state, props) => {
+  let result = inputPropertySelector(state, props)
```

Comment:
I'm seeing both `let` and `const` used in this PR and other spots in the editor, and it's not clear to me what convention is being followed here. I've picked up generally that the conventional wisdom is to use `const` consistently except when you **know** you need a variable that will be mutated, in which case, use `let`.

Reply:
That is my general practice as well to use `const` for immutable variables, changing this instance to a `let`.